package.json

```diff
@@ -13,28 +13,25 @@
   ],
   "types": "./lib/index.d.ts",
   "engines": {
-    "node": ">=8.12.0"
+    "node": ">=10.0"
```

Comment:
This is a change we discussed last month. `@svgr/cli@^5` dropped node 8 support. I didn't notice any issues with this change, and our Heroku pulls in node 15 anyway 😳 . We should probably enforce versions more strictly during cooldown so everyone is on the same version.